content/api/commands/hover.md

```diff
@@ -80,13 +80,14 @@ cy.get('.checkbox').check({ force: true })
 
 ### Add as a custom command
 
-Although Cypress does not have a built-in **cy.hover()**, it is a reserved
-command. Therefore, to add `cy.hover()` as a custom command;
-`Cypress.Commands.overwrite()` should be used instead of
-`Cypress.Commands.add()`.
+Although Cypress does not have a built-in `cy.hover()`, you can create your own
+custom command. While
+[Cypress.Commands.overwrite()](/api/cypress-api/custom-commands#Overwrite-Existing-Commands)
+still works, as of Cypress 10, we recommend using
+[Cypress.Commands.add()](/api/cypress-api/custom-commands).
```

Comment:
Can you use this text above your code example:

> Although Cypress does not have a built-in `cy.hover()` command, you can create your own custom command using [`Cypress.Commands.add()`](/api/cypress-api/custom-commands).

And then below that code example:

> Note that while `Cypress.Commands.add()` is the recommended way to define a custom `cy.hover()` command since Cypress 10.0, [`Cypress.Commands.overwrite()`](/api/cypress-api/custom-commands#Overwrite-Existing-Commands) will still work.

Followed by this code example:

```javascript
Cypress.Commands.overwrite('hover', (originalFn, ...otherArgs) => {})
```

Also, the callback to `Cypress.Commands.add()` doesn't receive the original function as the first argument like `Cypress.Commands.overwrite()` does, so any examples that do this

```js
Cypress.Commands.add('commandName', (originalFn, ...otherArgs) => {})
```

Need to be updated to something more like this

```js
Cypress.Commands.add('commandName', (...args) => {})
```

(Ie. your `hover` code example)